targets/esp8266/Makefile.esp8266

```diff
@@ -30,9 +30,21 @@ ESPTOOL    ?= /opt/Espressif/esptool-py/esptool.py
 
 # compile flags
 ESP_CFLAGS := -D__TARGET_ESP8266 -D__attr_always_inline___=
+
+MFORCE32 = `xtensa-lx106-elf-gcc --help=target | grep mforce-l32`
+
+ifneq ($(MFORCE32),)
+    # Your compiler supports the -mforce-l32 flag then 
+    # contants can be placed in ROM to free additional RAM
+    ESP_CFLAGS += -DJERRY_CONST_DATA="__attribute__((aligned(4))) __attribute__((section(".irom.text")))"
```

Comment:
I'm not an expert of GNU make, but the double quote usage is a bit odd to me here. Are you sure this will work as expected? Shouldn't be escaped the quotes around `.irom.text`?